packages/sdk/README.md

```diff
@@ -57,8 +57,8 @@ const messagePayload = createMessagePayload({
   domain: 'SIWT',
   address: 'tz1QpCttuR5qdQoo3FiT1cKwjqDhWUD21Vun',
   uri: 'https://siwt.xyz',
-  version: 1,
-  chainId: 'NetXdQprcVkpaWU',
+  version: '1',
```

Comment:
@royscheeren What version is this? Release version? Should we start thinking about this? https://github.com/StakeNow/SIWT/issues/18